src/tools/dev/scripts/build_visit

```diff
@@ -36,6 +36,7 @@
 #5  bv_<module>_depends_on [not implemented yet], also may be removed in favor of xml structure
 #6. bv_<module>_build builds the module
 
+export VISIT_MAJOR_VERSION=${VISIT_VERSION:-"3.1.0"}
```

Comment:
Why the change to 'VISIT_MAJOR_VERSION'?

Reply:
I only want to update the third party libraries with each major version, so I added VISIT_MAJOR_VERSION. I do see a typo though, that my testing wouldn't have caught. The line should be VISIT_MAJOR_VERSION=${VISIT_MAJOR_VERSION:-"3.1.0"}. I guess I could extract the major version from the VISIT_VERSION, but this seemed easier and I figure since the 2 lines are together, there shouldn't be a problem updating both.

Reply:
By major version, are you meaning just '3' or just '3.1'. Surely not '3.1.0'.

Reply:
I removed VISIT_MAJOR_VERSION since it is confusing. My new solution should also be more robust since it eliminates a potential maintenance issue in updating VISIT_MAJOR_VERSION.